jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -40,32 +42,49 @@ class LogGroupTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<Lis
 }
 
 class LogStreamDateColumn : ColumnInfo<LogStreamEntry, String>(message("general.time")) {
-    override fun valueOf(item: LogStreamEntry?): String? {
-        val timestamp = item?.timestamp ?: return null
-        return DateFormatUtil.getDateTimeFormat().format(timestamp)
-    }
+    private val renderer = ResizingDateColumnRenderer()
+    override fun valueOf(item: LogStreamEntry?): String? = item?.timestamp?.toString()
 
     override fun isCellEditable(item: LogStreamEntry?): Boolean = false
+    override fun getRenderer(item: LogStreamEntry?): TableCellRenderer? = renderer
 }
 
 class LogStreamMessageColumn : ColumnInfo<LogStreamEntry, String>(message("general.message")) {
-    override fun valueOf(item: LogStreamEntry?): String? = item?.message
-    override fun isCellEditable(item: LogStreamEntry?): Boolean = false
-}
-
-class WrappingLogStreamMessageColumn : ColumnInfo<LogStreamEntry, String>(message("general.message")) {
     private val renderer = WrappingLogStreamMessageRenderer()
+    fun wrap() {
+        renderer.wrap = true
+    }
+
+    fun unwrap() {
+        renderer.wrap = false
+    }
+
     override fun valueOf(item: LogStreamEntry?): String? = item?.message
     override fun isCellEditable(item: LogStreamEntry?): Boolean = false
     override fun getRenderer(item: LogStreamEntry?): TableCellRenderer? = renderer
 }
 
 private class WrappingLogStreamMessageRenderer : TableCellRenderer {
+    var wrap = false
+    // JBTextArea has a different font from JBLabel (the default in a table) so harvest the font off of it
+    private val font = JBLabel().font
```

Comment:
This needed with a JBTextArea?

Reply:
Tried using a JLabel and HTML but it doesn't adjust its preferred size, we need to reexamine how this is done later